src/Kernel/client/plotting.ts

```diff
@@ -1,12 +1,79 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-function importScript(url: string) {
-    let script = document.createElement("script");
-    script.setAttribute("src", url);
-    document.body.appendChild(script);
+// This is a bit of a hack needed to map the requireJS
+// call made by TypeScript onto the URL that Jupyter
+// makes our kernelspec available at.
+//
+// Using this hack, we can split the type import and
+// the runtime import apart, then glue them back
+// together using a declare global to solve TS2686.
+/// <amd-dependency path="/kernelspecs/iqsharp/chart.js" name="Chart" />
+import type * as ChartJs from "chart.js";
+declare global {
+    const Chart: typeof ChartJs;
 }
 
+export interface Complex {
+    Real: number;
+    Imag: number;
+    Magnitude: number;
+    Phase: number;
+};
+
+export interface DisplayableState {
+    n_qubits: number;
+    div_id: string;
+    amplitudes: Complex[] | null;
+};
+
+
 export function initializePlotting() {
-    importScript("https://cdn.jsdelivr.net/npm/chart.js@2.8.0");
-}
+    // importScript("https://cdn.jsdelivr.net/npm/chart.js@2.8.0");
+};
+
+
+
+export function createBarChart(element: HTMLCanvasElement, state: DisplayableState) {
+    var amps = state.amplitudes;
```

Comment:
I'd suggest keeping to `let` over `var`; I recognize that's controversial, but it'd at least be good to not mix `let` and `var` together.

Reply:
+1 to not using `var`. To go one step further, it looks like `amps` is not being mutated, so I would use `const`.

Reply:
changed var to let